### PR TITLE
Validate bulk import

### PR DESCRIPTION
**Pull request by jameschou93**

### CODE REVIEW

**Comment by jameschou93** [comment]
@trevordjones 
[trello card](https://trello.com/c/elCVlCKc/100-bug-fix-bulk-import-errors-out-nasty-when-not-a-supported-file-type)

File: app/views/documents/index.html.erb [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+    <% end %>
+
+    <% @documents_needed.each do |d| %>
+      <%= label_tag :category, d[0] %>
+      <%= file_field_tag :attachment %>
+    <% end %>
+
+    <input type="submit" name="attachments[attachment]">
+
+  <% end %>
```
Comment (trevordjones) [inline app/views/documents/index.html.erb]: You don't need to make any updates to this file. Please undo these changes

**Comment by trevordjones** [comment]
Also need to rebase and fix merge conflicts

**Comment by jameschou93** [comment]
Just rebased.
